admin/includes/modules/document_general/collect_info.php

```diff
@@ -210,7 +208,7 @@
     <div class="col-sm-9 col-md-6">
       <div class="date input-group" id="datepicker">
         <span class="input-group-addon datepicker_icon">
-          <i class="fa fa-calendar fa-lg"></i>
+          <i class="fa fa-calendar fa-lg">&nbsp;</i>

```

Comment:
Yes it is not "needed". Just nitpicking to get a clean result on the validator. Since it seems to make no difference visually if it is there or not, I thought why not.

Reply:
In my humble opinion the Validator is always wrong in this and a few other things.